libs/core-ui/src/lib/Cohort/ManualCohortManagement/CohortEditorPanelContent.tsx

```diff
@@ -128,6 +129,12 @@ export class CohortEditorPanelContent extends React.PureComponent<
           <Link className={styles.clearFilter} onClick={this.clearAllFilters}>
             {localization.Interpret.CohortEditor.clearAllFilters}
           </Link>
+          <div
```

Comment:
workspace-portal repo uses something like below as a component to announce. Very similar to this but as a component. Do you think this could be better for code readability and understandability?
https://msdata.visualstudio.com/Vienna/_git/workspace-portal?path=%2Fcommon%2Fcomponents%2Fsrc%2Fcustom%2Fannounce%2FAnnounce.tsx&_a=contents&version=GBmain
or Moving div to a common Announce file.

Reply:
Hi Vinu, it's actually different way of announcing the message, I can change to align with aml

Reply:
updated, thanks